### PR TITLE
[1LP][RFR]last_message update

### PR DESCRIPTION
**Pull request by lkhomenk**
Purpose or Intent
=================

__Fixing__  error logging for faulty provision requests. When we get  "Error" state of request, message requires some time to be updated
{{pytest: --long-running --use-provider complete cfme/tests/cloud/test_provisioning.py cfme/tests/infrastructure/test_provisioning_rest.py cfme/tests/infrastructure/test_provisioning.py cfme/tests/services/test_service_catalogs.py }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 0b8eff4e4be1abbb5e76f0c9e6779cd9d72d47d2:

`cfme/tests/infrastructure/test_provisioning_rest.py`:
- :bangbang: Line 99:13: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by lkhomenk** [comment]
It's not supposed to pass all PRT. I would even expect to get some provision failures.  But they should clearly tell what is wrong (depending on provider of course)
Rhos is the one to be checked first of all

**Comment by lkhomenk** [comment]
services/stacks need this - https://github.com/RedHatQE/cfme_tests/pull/6683

File: cfme/services/requests.py [hunk 1]
```diff
@@ -40,17 +40,27 @@ def __attrs_post_init__(self):
     def wait_for_request(self, num_sec=1800, delay=20):
         def _finished():
             self.rest.reload()
-            return self.rest.request_state.title() in self.REQUEST_FINISHED_STATES
+            return self.rest.request_state.title() in self.REQUEST_FINISHED_STATES and \
```
Comment (rlbabyuk) [inline cfme/services/requests.py]: Required: Please use parentheses for line continuation
The same for later occurrences

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by rlbabyuk** [comment]
There are some more places to correct

**Comment by izapolsk** [comment]
@lkhomenk , there is no PRT tests. Please figure out what's wrong and fix.

**Comment by lkhomenk** [comment]
PRT restarted. waiting for Wednesday

**Comment by lkhomenk** [comment]
last PRT shows all failed test as expected - we have final provision message

**Pull request merged**